app/views/articles/_form.html.erb

```diff
@@ -1,17 +1,26 @@
 <%= form_with model: @article do |form| %>
+  <% invalid = @article.errors.include?(:title) %>
   <div>
     <%= form.label :title %><br>
-    <%= form.text_field :title %>
-    <% @article.errors.full_messages_for(:title).each do |message| %>
-      <div><%= message %></div>
+    <%= form.text_field :title%>
+    <% if invalid %>
+      <div>
+        <% @article.errors.full_messages_for(:title).each do |error_message| %>
+          <%= error_message %>.
+        <% end %>
+      </div>
     <% end %>
   </div>
-
+  <% invalid = @article.errors.include?(:body) %>
```

Comment:
Every time when you need to assign a var in a view, know: you're doing a bad thing. You have `helpers` folder where you can put logic which should be placed in view. 

In your case will be better to just use `simple_form` gem. And I really recommend you to do it.